artemis-server/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
is this change accidental?

Reply:
Nope, has to be done with mockito if you want to override final class behaviours